libs/vkd3d/memory.c

```diff
@@ -429,18 +432,6 @@ static HRESULT vkd3d_memory_transfer_queue_flush_locked(struct vkd3d_memory_tran
     }
 
     /* Stall future submissions on other queues until the clear has finished */
-    memset(&timeline_info, 0, sizeof(timeline_info));
```

Comment:
This breaks 11on12 I think? Would D3D11 land know how to wait on these semaphores?

Reply:
what's the concern here exactly? we still submit waits to all queues later in the function and stall when locking the queue.

Reply:
Right, the waiters get flushed on any submission that is not VKD3D_SUBMISSION_WAIT, so the DRAIN command will end up waiting anyways, is that the assertion?

Reply:
yes, that's the idea. FWIW, our [11on12 test](https://github.com/doitsujin/dxvk-tests/blob/main/d3d11/d3d11_on_12.cpp) which is highly sensitive to this and exposed the memory initialization issue in the first place still works reliably.

Reply:
Ok, seems fine then.